app/src/ui/lib/list/list-row.tsx

```diff
@@ -221,7 +221,17 @@ export class ListRow extends React.Component<IListRowProps, {}> {
         onBlur={this.onBlur}
         onContextMenu={this.onContextMenu}
       >
-        {children}
+        {
+          // HACK: When we have an ariaLabel we need to make sure that the
+          // child elements are not exposed to the screen reader, otherwise
+          // VoiceOver will decide to read the children elements instead of the
+          // ariaLabel.
+          this.props.ariaLabel === undefined ? (
+            children
+          ) : (
+            <div aria-hidden={true}>{children}</div>
```

Comment:
It ignored the aria-label, but now that you ask… interestingly it still read the contents of the children elements 🤔 I will try again instead I did something wrong.

Reply:
Hmm it does work but according to [the documentation](https://www.w3.org/TR/wai-aria/#aria-expanded), it shouldn't 😂 Because the element with `aria-hidden=true` and its descendants should be ignored by assistive technologies.

Also looking at https://github.com/github/accessibility-audits/issues/4933#issuecomment-1747413748 I think we're expected to add `aria-hidden=true` to the contents, so I'm gonna stick to that